lib/darksky.rb

```diff
@@ -0,0 +1,33 @@
+require 'httparty'
+
+class Darksky
+  def self.darksky_info(coordinates)
+    darksky_url = "https://api.darksky.net/forecast/#{ENV['DARKSKY_API_KEY']}/#{coordinates}"
+    response = HTTParty.get(darksky_url)
+    if response.code == 200
+      return response
+    else
+      return "Sorry, your request could not be completed."
+    end
+  end
+
+  def self.weather_now(coordinates)
+    info = darksky_info(coordinates)
+    if info.is_a? String
```

Comment:
It seems like the only case where `darksy_info` can return a `String` is when you hit your `else` clause above. Since we do the same check and return the same value, why not just check the response code here?